test/functional/wallet_abandonconflict.py

```diff
@@ -120,6 +120,12 @@ def run_test(self):
         assert_equal(newbalance, balance + Decimal("30"))
         balance = newbalance
 
+        self.log.info("Check abandoned transactions in listsinceblock")
+        listsinceblock = self.nodes[0].listsinceblock()
+        txAB1_listsinceblock = [d for d in listsinceblock['transactions'] if d['txid'] == txAB1 and d['category'] == 'send']
+        for tx in txAB1_listsinceblock:
+            assert_equal(tx['abandoned'], True)
```

Comment:
Maybe also
```diff
         for tx in txAB1_listsinceblock:
             assert_equal(tx['abandoned'], True)
+            assert_equal(tx['confirmations'], 0)
+            assert_equal(tx['trusted'], False)
```

(apart from one assertion in `wallet_import_rescan.py` I think we currently have no test coverage of the `trusted` field, see #23139 that adds some)

Reply:
There are some fields that aren't covered by tests yet, I only added the abandoned field because of the proposal of the test. Would it be appropriate check these field (confirmations and trusted) here?

Reply:
Up to you! I'll re-review quickly if you update.

Reply:
Done!